doc/rtd/reference/network-config.rst

```diff
@@ -246,39 +246,46 @@ supplying an updated configuration in cloud-config. ::
 Network configuration tools
 ===========================
 
-``Cloud-init`` contains one tool used to test input/output conversion between
+``Cloud-init`` contains a command used to test input/output conversion between
 formats. The :file:`tools/net-convert.py` in the ``cloud-init`` source
-repository is helpful in examining expected output for a given input format.
+repository is helpful in examining expected output for a given input
+format. If running these commands from the cloud-init source directory,
+make sure to set the correct path ``PYTHON_PATH=.``
```

Comment:
Isn't this last sentence be true of any cloud-init CLI tool? I don't think it needs to be removed, but we might want to think of a more general place to put it.

Reply:
> Isn't this last sentence be true of any cloud-init CLI tool? 

Yes.

What was on my mind in adding this is that a few sub-commands stand out to me as "more likely" to be ran on a system that doesn't have cloud-init installed: schema, devel net-convert, devel render. This being one of them, I thought it was worth adding.